resources/providers/gcplib/inventory/provider.go

```diff
@@ -460,11 +493,23 @@ func getEcsGcpCloudData(ctx context.Context, crm *ResourceManagerWrapper, keys G
 
 func getOrganizationId(ancestors []string) string {
 	last := ancestors[len(ancestors)-1]
-	return strings.Split(last, "/")[1]
+	parts := strings.Split(last, "/") // organizations/1234567890
+
+	if parts[0] == "organizations" {
+		return parts[1]
+	}
+
+	return ""
 }
```

Comment:
this change was done to account for the case where there is only 1 ancestor - the org?

Reply:
It was done to account for a project without an organizational association.